keycloak/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# XXX: https://github.com/keycloak/keycloak/issues/17606 -- Keycloak v21+ removes SHA-1 support.
```

Comment:
Hwealth check can be set using non-modified keycloak image, see full example [here](https://github.com/NoumenaDigital/platform/blob/13a8c2ff2b8368abd9299003dcba82a48309f2d8/docker-compose.yml#L88)
```
    healthcheck:
      test: ["CMD-SHELL", "exec 3<>/dev/tcp/127.0.0.1/11000;echo -e 'GET /health/ready HTTP/1.1\r\nhost: http://localhost\r\nConnection: close\r\n\r\n' >&3;if [ $? -eq 0 ]; then echo 'Healthcheck Successful';exit 0;else echo 'Healthcheck Failed';exit 1;fi;"]
      interval: 1s
      retries: 60
```